casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -413,6 +412,9 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
     )
   }
 
+  private def blakeFromByteString[R](hash: StateHash): Blake2b256Hash =
```

Comment:
These two helpers belong in `Blake2b256Hash`

Reply:
Done.